core/templates/dev/head/services/SpeechSynthesisChunkerService.ts

```diff
@@ -170,21 +170,22 @@ export class SpeechSynthesisChunkerService {
     var elt = $('<div>' + html + '</div>');
     // Convert links into speakable text by extracting the readable value.
     elt.find('oppia-noninteractive-' + this.RTE_COMPONENT_NAMES.Link)
-      .replaceWith(() => {
+      .replaceWith(function() {
         if ((<HTMLElement><any> this).attributes[
           'text-with-value'] !== undefined) {
           return (<HTMLElement><any> this).attributes[
             'text-with-value'].textContent.replace(/&quot;/g, '');
         }
       });
 
+    var _this = this;
     // Convert LaTeX to speakable text.
     elt.find('oppia-noninteractive-' + this.RTE_COMPONENT_NAMES.Math)
-      .replaceWith(() => {
+      .replaceWith(function() {
         if (
           (<HTMLElement><any> this).attributes[
             'raw_latex-with-value'] !== undefined) {
-          return this._formatLatexToSpeakableText(
+          return _this._formatLatexToSpeakableText(
```

Comment:
Instead of using `_this`, I can re-define `_formatLatexToSpeakableText` on top (line 181) and use it here, let me know if you want me to do that?